apps/builder/components/social-option-item/social-option-item.tsx

```diff
@@ -5,6 +5,7 @@ import type { Transform } from '@dnd-kit/utilities'
 import classNames from 'classnames'
 
 import { SocialProvider } from '@reown/appkit-core'
+import '@reown/appkit-ui/jsx'
```

Comment:
As I see `/jsx` path is exporting `JSXTypeUtil` which is exporting the type definition for all components. let's rename it as `@reown/appkit-ui/types`, which is more understandable?

Reply:
Also why we are importing this only on `social-option-item.tsx`? Not needed for others?

Reply:
I'm not sure why we are calling it as `jsx`,  `jsx` is coming from React terminology, that it is a syntax extension for JavaScript used with React. But on the other hand we are using Web Components as base.

I understand we might need this type definition to make the React happy with our component types;
```js
namespace JSX {
    ...
  }
```

But jsx shouldn't be in our main naming keyword imo

Reply:
I mean literally the utility is to add jsx definitions to the global object. wyt if I include in the react/vue exports directly?